site/metrics.md

```diff
@@ -20,6 +20,15 @@ exposed:
 * `weave_dns_entries` - Number of DNS entries.
 * `weave_flows` - Number of FastDP flows.
 
+#### Publish Router Metrics Endpoint
+
+By default weave only listens on it's local interface to serve metrics. 
```

Comment:
This is true when started from the `weave` script, but not true when started from Kubernetes via `weave-kube`.

Suggest "By default, when started via `weave launch`"